packages/shared/routes/dashboard/settings/views/advanced/Advanced.svelte

```diff
@@ -33,12 +33,12 @@
 
 <div>
     {#each settings as { component, childRoute, requireLogin, requireLedger }, index}
-        {#if (!requireLogin || (requireLogin && $loggedIn)) && (!requireLedger || (requireLedger && $isLedgerProfile))}
+        {#if (!requireLogin || (requireLogin && $loggedIn)) && (!requireLedger || (requireLedger && $isLedgerProfile)) && $mobile && $settingsChildRoute === childRoute}
             <section id={childRoute} class="w-full sm:w-3/4">
                 <svelte:component this={component} id={childRoute} />
             </section>
             {#if index < settings.length - 1}
-                <HR classes="pb-5 mt-5 justify-center" />
+                <HR classes="pb-5 mt-5 justify-center hidden md:block" />
```

Comment:
Line 36: Good catch! :D just fixed with correct expression.
Line 41: It's for both mobile and desktop, could be written as:
```svelte
<HR classes="pb-5 mt-5 justify-center {!$mobile && 'hidden block'}" />
```
but I think it's better as is made, because it's avoinding the conditional `$mobile` store.

Reply:
Yes, even though the if statement is a bit hard to read, I guess it`s fine now 👍